src/clarity/forms/_forms.clarity.scss

```diff
@@ -434,6 +433,23 @@ $clr-custom-checkbox-radio-top: ($clr_baselineRem_1 - $clr-checkbox-radio-height
             display: inline-block;
         }
 
+        input[type="checkbox"]:indeterminate + label::before {
```

Comment:
My assumption is that these styles work outside of the angular code. If that is the case, can we make sure to put a static example on the website so that people who aren't using Clarity-NG will know how to use these?

And if it isn't the case, can we make sure that it is?

Please validate that this selector will work in IE10-11. I have a bad feeling about it...